docs/reference/experiment-config.txt

```diff
@@ -472,21 +467,21 @@ created with the Determined CLI. It may contain the following fields:
 .. _batches-per-step:
 
 - ``batches_per_step``: Specifies the number of batches in a single
-  training step. As discussed above, Determined divides the training of a
-  single trial into a sequence of steps; each step corresponds to a
-  certain number of model updates. Therefore, this configuration
+  training workload. As discussed above, Determined divides the training of a
+  single trial into a sequence of training workloads; each workload corresponds
+  to a certain number of model updates. Therefore, this configuration
   parameter can be used to control how long a trial is trained at a
   single agent:
 
-  - Doing more work in a step allows per-step overheads (such as
+  - Doing more work in a workload allows per-workload overheads (such as
     downloading training data) to be amortized over more training
-    work. However, if the step size is too large, a single trial might
+    work. However, if the workload size is too large, a single trial might
```

Comment:
"if the workload size is too large, a single trial" -> "if the size of a single workload is too large, a trial"